README.md

```diff
@@ -1,138 +1,52 @@
 # Skyline DataMiner Deploy Action
 
-This action builds a DataMiner Artifact from your Automation Script solution and deploys it to your cloud-connected DataMiner System. The action will wait until the deployment is finished, with a configurable timeout. At present, only DataMiner Automation Script solutions created by DIS are supported.
+> [!IMPORTANT]
+> This GitHub action has been deprecated and is replaced by .NET tools which makes it easier to create workflows/pipelines/... outside GitHub and still be able to deploy package to DataMiner.
+>
+> The old readme file can be found here: [old readme](oldReadme.md)
 
-The action consists of 2 stages: `Upload` and `Deploy`.
+## Migration to .NET Tools
```

Comment:
```suggestion
## Transition to .NET Tools

Our workflow has evolved from relying solely on GitHub actions to embracing the versatility of .NET tools. This transition offers enhanced flexibility, enabling seamless integration across various widely utilized platforms such as GitHub, GitLab, Azure DevOps, and more.

Previously, the GitHub Action encompassed a bundled approach, performing three distinct tasks:

1. Generating a package
2. Uploading the package to the Catalog
3. Deploying the package to a DataMiner

However, this bundled functionality didn't always align with the specific needs of our users. In response, we have modularized these tasks into individual .NET tools, allowing for tailored usage based on the unique requirements of each scenario.

Below, we present an example detailing the migration process from the GitHub action:
```

Reply:
Note, you'll need to remove the other text before applying the suggestion or you'll have the new and old text.